tests/test_animation.py

```diff
@@ -145,9 +145,11 @@ def test_send_animation_url_file(self, bot, chat_id, animation):
         assert message.animation.file_unique_id != ''
 
         assert message.animation.duration == animation.duration
-        assert message.animation.file_name == animation.file_name
+        assert message.animation.file_name.startswith(
+            'game.gif'
+        ) == animation.file_name.startswith('game.gif')
```

Comment:
what does  this do? do the file names differ?

Reply:
one is `game.gif.mp4` other one is `game.gif.webm`

Reply:
wtf. what happens if you replace the bitly link with `'https://python-telegram-bot.org/static/testfiles/game.gif?no=cache'` ? (the parameter should avoid caching)

Reply:
same result

Reply:
geee … okay let's keep it then. You can merge after removing the file size test.
btw deepsource promised to fix the failing tests soonish